doc/source/whatsnew/v0.20.0.txt

```diff
@@ -83,3 +83,10 @@ Performance Improvements
 
 Bug Fixes
 ~~~~~~~~~
+
+.. _whatsnew_0200.gbq:
+
+Google BigQuery Enhancements
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+- The :func:`read_gbq` method now allows query configuration preferences
```

Comment:
@jreback 
Sorry, but I have some newbie questions
if there is no issue for that feature do I need to create one now? And do I must add tiny example to ```whatsnew``` file or to ```issue``` file?